src/components/Attachments/AttachmentView/AttachmentViewPdf/propTypes.js

```diff
@@ -8,12 +8,14 @@ const attachmentViewPdfPropTypes = {
     encryptedSourceUrl: PropTypes.string.isRequired,
     onToggleKeyboard: PropTypes.func.isRequired,
     onLoadComplete: PropTypes.func.isRequired,
+    errorLabelStyles: PropTypes.arrayOf(PropTypes.object),
```

Comment:
```suggestion
    errorLabelStyles: stylePropTypes,
```

Reply:
in both files `errorLabelStyles` prop type needs a comment explaining what it's used for